source/smart-contracts/general/develop-contracts.rst

```diff
@@ -42,7 +42,7 @@ compiled to Wasm.
 To obtain correct exports, the `crate-type` attribute must be set to
 ``["cdylib", "rlib"]`` in the manifest file:
 
-.. code-block::
+.. code-block:: console
```

Comment:
I'm not sure if this should be a console block or just a plain code-block.

Reply:
Makes sense, changed it to `text` now.